tarantool_test.go

```diff
@@ -2037,27 +2041,36 @@ func TestSchema(t *testing.T) {
 	}
 
 	var rSpaceNo, rIndexNo uint32
-	rSpaceNo, rIndexNo, err = schema.ResolveSpaceIndex(616, 3)
-	if err != nil || rSpaceNo != 616 || rIndexNo != 3 {
-		t.Errorf("numeric space and index params not resolved as-is")
+	rSpaceNo, err = schema.ResolveSpace(616)
+	if err != nil || rSpaceNo != 616 {
+		t.Errorf("numeric space param not resolved as-is")
 	}
-	rSpaceNo, _, err = schema.ResolveSpaceIndex(616, nil)
+	rIndexNo, err = schema.ResolveIndex(3, rSpaceNo)
+	if err != nil || rIndexNo != 3 {
+		t.Errorf("numeric index param not resolved as-is")
+	}
+	rSpaceNo, err = schema.ResolveSpace(616)
 	if err != nil || rSpaceNo != 616 {
 		t.Errorf("numeric space param not resolved as-is")
 	}
-	rSpaceNo, rIndexNo, err = schema.ResolveSpaceIndex("schematest", "secondary")
-	if err != nil || rSpaceNo != 616 || rIndexNo != 3 {
-		t.Errorf("symbolic space and index params not resolved")
+	rSpaceNo, err = schema.ResolveSpace("schematest")
+	if err != nil || rSpaceNo != 616 {
+		t.Errorf("symbolic space param not resolved")
+	}
+	rIndexNo, err = schema.ResolveIndex("secondary", rSpaceNo)
+	if err != nil || rIndexNo != 3 {
+		t.Errorf("symbolic index param not resolved")
 	}
-	rSpaceNo, _, err = schema.ResolveSpaceIndex("schematest", nil)
+	rSpaceNo, err = schema.ResolveSpace("schematest")
 	if err != nil || rSpaceNo != 616 {
 		t.Errorf("symbolic space param not resolved")
 	}
-	_, _, err = schema.ResolveSpaceIndex("schematest22", "secondary")
+	_, err = schema.ResolveSpace("schematest22")
 	if err == nil {
 		t.Errorf("ResolveSpaceIndex didn't returned error with not existing space name")
 	}
-	_, _, err = schema.ResolveSpaceIndex("schematest", "secondary22")
+	rSpaceNo, _ = schema.ResolveSpace("schematest")
+	_, err = schema.ResolveIndex("secondary22", rSpaceNo)
```

Comment:
There are no integration tests for `schema.NamesUseSupported()`.

Reply:
New test added.